### DISCUSSION TITLE
Request: Use Inspect settings in exports

### DISCUSSION THREAD

**Discussion by adamwilt** [videovillage/community#134] [Ideas]
At present, saving a screenshot and exporting a clip apply the LUT settings in the Color window and the sizing settings in the Sizing window, but the Inspect settings (gamma, exposure, color channel, etc.) are ignored. 

Being able to use the Inspect settings, especially gamma and exposure, for exports would (a) make exports truly WYSIWYG, and (b) makes it easier to generate "dailies". 

Often I want to fake a quick 'n' dirty log-to-709 look without a LUT, and/or fix exposure a bit, and being able to bake exposure and gamma into the exports is needed. Sometimes I want to export a zoomed-in detail, too. And it's rare, but I can see cases where I'd want to be able to export just one color channel (or luma, or alpha).

**Comment by hotchkiss** [comment]
Would love this feature as well. I was looking at a VFX shot submission today and turned the gamma up to see a detail better, and it would have been great to have been able to export with those settings applied.
